pep-0497-externals.txt

```diff
@@ -0,0 +1,252 @@
+PEP: 497
+Title: Enhanced handling of external requirements
+Version: $Revision$
+Last-Modified: $Date$
+Author: Tennessee Leeuwenburg <tleeuwenburg@gmail.com>,
+
+Status: Draft
+Type: Standards Track
+Python-Version 3.6
+Content-Type: text/x-rst
+Created: 03-Aug-2015
+Post-History: 03-Aug-2015
+
+
+Abstract
+========
+
+The canonical method for installing packages is to use pip, whether that is
+referencing PyPI or a local repository. A limitation of the current install
+ecosystem is the relationship with operating-system level dependencies. This
+can result in compilation failures or execution failures. Package maintainers
+may wish to specify the requirements to avoid this issue, and users may
+appreciate earlier stopping and more informative messages about their system.
```

Comment:
Might read more smoothly as 'appreciate earlier and more informative error messages.'